### ISSUE TITLE
Bug when reconnecting to `top`/`ytop`

### ISSUE THREAD

**Issue by austinjones** [austinjones/tab-rs#135]
Reconnecting to a running `top` or `ytop` session does not work.

**Reproduction**
```
tab top
top
ctrl-W
tab top
```

**Expected Behavior**
The session resumes with the latest stdout.

**Actual Behavior**
No stdout is printed, and `ctrl-C`/`ctrl-Z` do nothing.

**Comment by austinjones** [comment]
`ctrl-C` does actually kill the process.

This is caused by frequent stdout chunks, faster than the scrollback chunks arrive.  I was able to reproduce with:
```
while true; date; sleep 0.1; end
```

**Comment by austinjones** [comment]
Resolved by #141 and #142
